### PR TITLE
Move builds to makeincl

### PR DESCRIPTION
**Pull request by kkuzmin**
Replace wget with curl

### CODE REVIEW

**Comment by kkuzmin** [comment]
test it

File: ps_spec.yml [hunk 1]
```diff
@@ -3,19 +3,18 @@ service_name: haproxy-centos
...
       ALPS_SERVICE_VERSION: 1.5.10
     commands:
       - echo AMI build started on `date`
-      - source ~/.bashrc
       - git config core.warnAmbiguousRefs false
       - make package
       - mkdir /artifact_folder
-      - rm -rf tmp/RPMS/x86_64/*debuginfo*
-      - cp tmp/RPMS/x86_64/*.rpm /artifact_folder/
+      - rm -rf tmp/RPMS/$(arch)/*debuginfo*
```
Comment (motobob) [inline ps_spec.yml:16]: is this even set?

**Review by motobob — APPROVED** [review]
aside from $(arch), rest looks good

File: ps_spec.yml (hunk 1 above)
Comment (kkuzmin) [reply ps_spec.yml:16]: that is a command like utility `arch`

**Pull request merged**